lib/rules/no-control-regex.js

```diff
@@ -83,8 +96,10 @@ module.exports = {
                 if (regex) {
                     const computedValue = regex.toString();
 
-                    if (hasControlCharacters(computedValue)) {
-                        context.report(node, "Unexpected control character in regular expression.");
+                    const controlCharacters = getControlCharacters(computedValue);
+
+                    if (controlCharacters.length > 0) {
+                        context.report(node, "Unexpected control character(s) in regular expression: " + controlCharacters.join(", ") + ".");
```

Comment:
In the editor integrations the character in question will just be highlighted, no need to grep anything. Also, if the same character shows up more then one time in a string, you will have trouble finding it with just one error message.